content/SCALE/SCALECLIReference/Account/CLIgroup.md

```diff
@@ -10,8 +10,221 @@ tags:
 
 {{< toc >}}
 
-
 {{< include file="/_includes/CLI/CLIGuideWIP.md" type="page" >}}
 
+The **group** namespace contains 8 commands and is based on functions found in the SCALE API and web UI. It provides access to group account creation, configuration, and management.
+
+You can enter commands from the main CLI prompt or from the system namespace prompt.
+
+## Create Command
+The `create` command creates a new group.
+
+{{< include file="content/_includes/CLI/AccountGroupCreateExample.md" type="page" >}}
+
+### Create Interactive Arguments Editor
+
+The `create --` option opens an **interactive arguments editor**.
+
+{{< expand "Using the create Interactive Arguments Editor" "v" >}}
+{{< trueimage src="/images/SCALE/CLI/Account/GroupCreateInteractiveArgumentsEditor.png" alt="Interactive Arguments Editor" id="1: Interactive Arguments Editor" >}}
```

Comment:
is this the TUI for the create command? if yes, you can include that in the hover and caption text. I'm wondering if we should make this an expand so the reader can make the choice to see this content after the heading verses having it display by default. Thoughts?

Also, I think the content following this is better suited to that global content we talked about, then you can focus on the specifics of using the TUI for the create command in this article.